changelog/unreleased/pr-20996.toml

```diff
@@ -0,0 +1,5 @@
+type = "a"
+message = "Add auto refresh to events page"
```

Comment:
This sounds like the events list is now refreshing automatically when you open it.